packages/sdk/src/sdk/config/types.ts

```diff
@@ -29,9 +31,11 @@ export type SdkServicesConfig = {
   ethereum: {
     rpcEndpoint: string
     addresses: {
-      serviceTypeManagerAddress: `0x${string}`
-      serviceProviderFactoryAddress: `0x${string}`
-      ethRewardsManagerAddress: `0x${string}`
+      serviceTypeManagerAddress: Hex
```

Comment:
I both like this and also get confused because there is another library somewhere in our deps that has a `hex` type that means a byte buffer 😅